### PR TITLE
Remove unused functions from application window

### LINKED ISSUE #16326: Remove obsolete or unused functions in MantidPlot
(Found during work on #16285)

Some functions in `ApplicationWindow.cpp` do not seem to get called anywhere, and are probably obsolete - for example, [showStandAloneHelp()](https://github.com/mantidproject/mantid/blob/7df701a7d7f3eebb8c155f510f93455f3048a057/MantidPlot/src/ApplicationWindow.cpp#L10460-L10512) which was originally meant to show the QtiPlot manual. (In this and other places, the only call to the function is commented out.)

This task is to find dead code that is no longer used and remove it.

See also: *View > Preferences > General > File Locations*

### PR DESCRIPTION
**Pull request by lottiegreenwood**
Removed code that had been commented out in ApplicationWindow, and some code that wasn't being called

**To test:**

Review code

Fixes #16326.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by KarlPalmen** [comment]
In commit message, please put the 're' or 'ref' nearer the end. It makes code reviewing easier, especially if there are a large number of commits.

**Comment by KarlPalmen** [comment]
Code review is OK, but suggests a build and run is necessary to check that nothing vital has been removed.

**Comment by KarlPalmen** [comment]
Works OK :shipit:

**Comment by FedeMPouzols** [comment]
I'm afraid this branch has developed a merge conflict, probably with the other many branches that have updated ApplicationWindow to remove Qt3Support warnings. Hopefully it's not a big one.

File: MantidPlot/src/ApplicationWindow.cpp [hunk 1]
```diff
@@ -13535,8 +13409,8 @@ void ApplicationWindow::translateActionsStrings() {
   actionAutoCorrelate->setMenuText(tr("&Autocorrelate"));
   actionConvolute->setMenuText(tr("&Convolute"));
   actionDeconvolute->setMenuText(tr("&Deconvolute"));
-  actionTranslateHor->setMenuText(tr("&Horizontal"));
-  actionTranslateVert->setMenuText(tr("&Vertical"));
+  //actionTranslateHor->setMenuText(tr("&Horizontal"));
```
Comment (martyngigg) [inline MantidPlot/src/ApplicationWindow.cpp]: Is there a reason we are adding back commented-out code?

File: MantidPlot/src/ApplicationWindow.h [hunk 2]
```diff
@@ -652,9 +652,9 @@ public slots:
   void fitSigmoidal();
   void fitGauss();
   void fitLorentz();
-  void fitMultiPeak(int profile);
-  void fitMultiPeakGauss();
-  void fitMultiPeakLorentz();
+  //void fitMultiPeak(int profile);
```
Comment (martyngigg) [inline MantidPlot/src/ApplicationWindow.h]: Is there a reason we are adding back commented-out code?

**Comment by martyngigg** [comment]
Confirmed that the latest set of changes have only removed commented out code.

The pull request shows many changes because of the recent merge of the release branch and I'm assuming that the PR comparison compares the changes between when the pull request was created? The [current compare view](https://github.com/mantidproject/mantid/compare/5b9bd49bfcd30d3ccb1d14fe8b093148c6205485...16326_Remove_unused_functions_from_applicationWindow), i.e. with the master at the time of writing, shows a much more sensible set of changes.

**Pull request merged**